Legacy/real_time/arduino_src/radio_buggy_mega/Makefile

```diff
@@ -467,51 +494,46 @@ extcoff: $(TARGET).elf
 
 
 # Link: create ELF output file from object files.
-.SECONDARY : $(TARGET).elf

```

Comment:
Even though it seems useless, should we apply the moto: don't fix what ain't broken?